src/vm.rs

```diff
@@ -9,6 +9,7 @@ use hermit_entry::{
 };
 use log::{error, warn};
 use thiserror::Error;
+use uhyve_interface::{parameters::*, Hypercall, HypercallAddress, MAX_ARGC_ENVC};
```

Comment:
Personal preference: Expand the glob import.

Reply:
Usually yes, but we include everything from this module: I'd keep it as it is annoying enough to do all of these fixup commits.